Gems/ROS2/Code/Source/VehicleDynamics/ManualControlEventHandler.h

```diff
@@ -71,6 +71,14 @@ namespace ROS2::VehicleDynamics
                         ownerEntity, &VehicleInputControlRequests::SetTargetLinearSpeedFraction, inputValue);
                 }));
 
+            m_eventHandlers.push_back(ManualControlSingleEventHandler(
+                "rotate",
+                [ownerEntity](float inputValue)
```

Comment:
nit: Might be worth at this time to renaming the variable to `ownerEntityId` for clarity (so it's not mistaken as an `Entity*`

Reply:
Good, adjusted.